www/less/misc.less

```diff
@@ -82,3 +82,26 @@ a{
 * {
     -webkit-touch-callout: none !important;
 }
+
+//updating typography for h2 and h3
+.content h2 {
```

Comment:
Probably want to scope these more tightly to h2's which are definitely section headers... i.e. `h2.section_heading` instead of `.content h2`. Same below and for h3's.